draft-ietf-tls-dtls13.md

```diff
@@ -681,7 +694,9 @@ receiver MUST verify that the record contains a sequence number that
 does not duplicate the sequence number of any other record received
 during the lifetime of the session. This check SHOULD happen after
 deprotecting the packet; otherwise the packet discard might itself
-serve as a timing channel for the sequence number.
+serve as a timing channel for the sequence number. Note that decompressing
+the record number is still a potential timing channel, though a weaker
+one.
```

Comment:
Weaker seems - for want of a better word - weak.  The consequence here is that you expose some information about the sequence number. That doesn't provide an attacker much advantage in terms of assisting in key recovery because the key is independent of other keys and the protected information is very low entropy anyway.